app/src/main/res/layout-land/pin_password_activity.xml

```diff
@@ -69,7 +69,7 @@
       android:layout_marginTop="8dp"
       android:cursorVisible="true"
       android:fontFamily="sans-serif"
-      android:inputType="@{viewModel.showPassword ? InputType.TYPE_CLASS_NUMBER : (InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD)}"
+      android:inputType="phone"
```

Comment:
as per my comment in the issue, we are using the library version because there is a new method introduced in the library itself to resolve this issue. 
Please check this comment by the author of the library. https://github.com/ChaosLeung/PinView/issues/55#issuecomment-691789973